### ISSUE TITLE
source cppreference.com broken

### ISSUE THREAD

**Issue by SGSSGene** [aitjcize/cppman#86]
Hi, thanks for this awesome tool. I love it!
But I would like to use cppreference.com as a source (this work in the past).

Currently it doesn't work. (I tried the newest version from github).
```
$ ./bin/cppman -s cppreference.com
$ ./bin/cppman std::vector
error: bad escape \E at position 4
```
Thank you

**Comment by aitjcize** [comment]
Hi @SGSSGene ,

I'm glad that you like the project. The problem you described seems to be the problem I fixed in the 0.4.9 release, can you try clear the cache (`cppman -C`) first and try again? Also make sure you are using v0.4.9.

**Comment by SGSSGene** [comment]
I checked 0.4.9, but still didn't work.
The issue is that the fix in 0.4.9 was only for cplusplus.com and not for cppreference.com. I adopted your fixes for cppreference.com and made a PR: https://github.com/aitjcize/cppman/pull/87

**Comment by SGSSGene** [comment]
fixed in #87
